pkg/controller/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -74,6 +75,34 @@ func prepareReconcile(t *testing.T, requestName string, httpTestClient *fakeHTTP
 	return r, reconcile.Request{NamespacedName: test.NamespacedName(test.HostOperatorNs, requestName)}, fakeClient
 }
 
+func prepareReconcileWithStatusConditions(t *testing.T, requestName string, httpTestClient *fakeHTTPClient,
+	getMemberClustersFunc func(fakeClient client.Client) cluster.GetMemberClustersFunc, conditions []toolchainv1alpha1.Condition, initObjs ...runtime.Object) (*ReconcileToolchainStatus, reconcile.Request, *test.FakeClient) {
+	reconciler, req, fakeClient := prepareReconcile(t, requestName, newResponseGood(),
+		newGetMemberClustersFuncReady, initObjs...)
```

Comment:
just a detail - can be on a single line: 
```suggestion
	reconciler, req, fakeClient := prepareReconcile(t, requestName, newResponseGood(), newGetMemberClustersFuncReady, initObjs...)
```